apps/site/assets/js/algolia-autocomplete.js

```diff
@@ -6,23 +6,22 @@ import * as QueryHelpers from "../ts/helpers/query";
 /* eslint-disable class-methods-use-this */
 
 export default class AlgoliaAutocomplete {
-  constructor({ id, selectors, indices, parent }) {
+  constructor({ id, selectors, indices, parent, containerEl = null }) {
```

Comment:
It would be good to test that `AlgoliaAutocomplete` can optionally accept `containerEl` as a property, as well as testing how that affects its behavior. (As a unit test, that is. I see that it is triggered indirectly via `trip-plan_test.js` as well.)